tutorial/index.jade

```diff
@@ -160,10 +158,10 @@
       Much as we did for our Authy client, we create a single instance of the
       Twilio REST API helper, called `client` in this example.
 
-      Then all we need to do to send an SMS to the user's phone is use the built
-      in `send()`. Notice that we'll use the user's `fullNumber()` to make sure
-      we support international numbers. The `getFullPhoneNumber()` method in the
-      `User` model simply returns a combination of the `countryCode` and
+      Then all we need to do to is send an SMS to the user's phone. This is done
+      using the `send()` build. Notice that we'll use the user's `fullNumber()` to
```

Comment:
Could you please change _build_ by _method_? In this context _build_ does not make any sense.